src/seedu/addressbook/Main.java

```diff
@@ -83,14 +81,30 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
+            //List all persons automatically after an add or delete command
             command = new Parser().parseCommand(userCommandText);
             CommandResult result = executeCommand(command);
             recordResult(result);
             ui.showResultToUser(result);
 
+            if(command instanceof AddCommand || command instanceof DeleteCommand) {
+                listTheirDetails();
+            }
+
         } while (!ExitCommand.isExit(command));
     }
 
+    /*
+     * Executes list command when add or delete command is called.
```

Comment:
You can remove `when add or delete command is called` as it might be used in other cases.